control/controlclient/direct.go

```diff
@@ -365,6 +367,15 @@ func (c *Direct) doLogin(ctx context.Context, t *tailcfg.Oauth2Token, flags Logi
 	request.Auth.Provider = persist.Provider
 	request.Auth.LoginName = persist.LoginName
 	request.Auth.AuthKey = authKey
+	err = signRegisterRequest(&request, c.serverURL, c.serverKey, c.machinePrivKey.Public())
+	switch err {
+	case nil:
+		// Success
+	case errCertificateNotConfigured, errNoCertStore:
+		// The feature is disabled, no need to log
```

Comment:
do you want to nil out RegisterRequest.Timestamp (from line 361) in this case, to not send it to the server?